nearlib/signing/account_info.js

```diff
@@ -0,0 +1,79 @@
+const KeyPair = require('./key_pair');
+
+/**
+ * Standard format for storing account info in json
+ */
+class AccountInfo {
```

Comment:
I'm still not convinced either this or `KeyPair` should be a class.

Reply:
time estimate of refactor of removing KeyPair class in favor of json object: 1 day. Would like someone else to make a call on ROI of this.

Reply:
I'm not saying it should be done as part of this PR (re: KeyPair), cause it wasn't added / touched in this PR.

However generally it works best (especially in composability) in JS when you keep simple data objects and helper functions separate. 

If we were starting from scratch what would be better ROI:
1. writing a 3-line method to throw on invalid JSON
2. writing 50-line wrapper class

I'm not gonna block this PR on this specific issue anymore, but I think we need to agree on general design approach moving forward.

Reply:
we have more than throwing on invalid json though:
* how to generate the json object (will be used by any place that can save keys as json)
* generate a standard file name
I think that this warrants a class

Reply:
generating file name belongs to key / account store IMO, as generally account info doesn't have to be in file (e.g. I can have multiple accounts stored in DB / localStorage / etc).

> how to generate the json object (will be used by any place that can save keys as json)

if goal is to minimize code, generating json can be just JSON.stringify(object) at any place that can save keys as json.

Reply:
I don't trust every place to generate the correct json object though. also if we version/change file formats having this json generating code everywhere will be a nightmare

Reply:
> also if we version/change file formats having this json generating code everywhere will be a nightmare

it doesn't really have to be nightmare

let's assume you have helper:

```
function parseAccount(jsonObject) {
   // TODO: Validate passed json here
  return jsonObject;
}
```

Assume all code uses `parseAccount(JSON.parse(jsonString))` and `JSON.stringify()`. If at any point you actually want to change format – you can change `parseAccount` to return object with `toJSON` method which does necessary conversion.

Reply:
Now you are building the thing that I already (after putting a day's thought into it AND building tests for it) in slightly different style. Can we just go with the existing solution rather than spending another day on moving code around?

Reply:
As I say, I do not plan to block PR for this issue anymore. However I'd really prefer we don't generate more entities in similar way when moving forward.

Reply:
It will be hard to ensure that every line of javascript written going forward will match your ideal code style 100% going forward. There has to be room for working with different styles.